paint/randomness-collective-flip/src/lib.rs

```diff
@@ -239,11 +233,12 @@ mod tests {
 
 			setup_blocks(81);
 
-			let random_material = CollectiveFlip::random_material();
+			let random_material_0 = CollectiveFlip::random_material(0);
+			let random_material_1 = CollectiveFlip::random_material(1);
 
-			assert_eq!(random_material.len(), 81);
-			assert_ne!(random_material[0], random_material[1]);
-			assert_eq!(random_material[0], genesis_hash);
+			// assert_eq!(random_material.len(), 81);
```

Comment:
Unless you're going to be re-adding these checks later, I'd just remove the line.

Reply:
Yes, the plan is to re-add it.